internal/play/play_test.go

```diff
@@ -33,9 +33,9 @@ func TestPlay(t *testing.T) {
 
 	api.On("GetPlayback").Return(playback1, nil).Once()
 	api.On("GetPlayback").Return(playback2, nil).Once()
-	api.On("Play", "", []string(nil)).Return(nil)
+	api.On("Play", "","", []string(nil)).Return(nil)
 
-	status, err := Play(api, "", "")
+	status, err := Play(api, "", "", "")
```

Comment:
Yikes, this is a lot of empty strings... I'm going to get rid of the `deviceID` string in a separate PR, since it was a failed experiment.